libminifi/test/rocksdb-tests/ContentSessionTests.cpp

```diff
@@ -56,11 +56,10 @@ const std::shared_ptr<minifi::io::BaseStream>& operator>>(const std::shared_ptr<
   str = "";
   uint8_t buffer[4096]{};
   while (true) {
-    size_t ret = stream->read(buffer, sizeof(buffer));
+    const auto ret = stream->read(buffer, sizeof(buffer));
     REQUIRE(ret >= 0);
```

Comment:
```suggestion
    REQUIRE(!minifi::io::isError(ret));
```

Reply:
We need both checks, and the new one is one line below. Maybe I misunderstood your point?

Reply:
I think I get it: `ret >= 0` is always true, I'll remove it.

Reply:
somehow I didn't see the other check below it -- but yes, this can be removed

Reply:
done in 141917b